src/reduxForm.js

```diff
@@ -255,6 +255,7 @@ const createReduxForm =
 
           render() {
             // remove some redux-form config-only props
+            /* eslint-disable no-unused-vars */
```

Comment:
This won't be addressed until _Rest/Spread Properties_ would progress to higher stage.
See https://github.com/eslint/eslint/issues/4880#issuecomment-169860909.